### DISCUSSION TITLE
SC解包时addon和lobby依旧出错

### DISCUSSION THREAD

**Discussion by mokeling** [GpointChen/FFXIVChnTextPatch-GP#353] [General]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/56252068/331af4c4-1d1b-4991-a43c-b0beee51b2e7)

显示长度不够，请问现在有办法解决了吗

**Comment by mokeling** [comment]
或者是否有其他方式能获得hex格式的解包文件

**Comment by GpointChen** [comment]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/34565875/ec6811e2-7ba1-4cba-95da-b8b65ea5bb9b)

我這邊addon跟lobby是正常的

試著用最新版的definition看看？

**Comment by mokeling** [comment]
我用的就是最新版definition，是汉化之后解包的，汉化前解包确实没问题，主要是想看一下有哪些地方是没汉化的想自己改一下

**Comment by GpointChen** [comment]
了解，那可能是我修改後有指令碼出問題。

我晚點檢查一下。

**Comment by mokeling** [comment]
好的，感谢大大

**Comment by GpointChen** [comment]
測試了一下繁體版沒有問題，簡體版的用今天剛修改過的版本繁轉簡後也沒有問題

你用下面那個檔案測試看看？

[rawexd-zhcn-temp.zip](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/11565233/rawexd-zhcn-temp.zip)

**Comment by mokeling** [comment]
还是不行。。。会不会是我编译的sc有问题，大大可以发我一份试试吗(；′⌒`)

**Comment by mokeling** [comment]
用原版的sc也不行。。。应该也不是编译的问题，搞不懂了

**Comment by GpointChen** [comment]
我剛才重試反而跳出了別的exception

我有空找一下問題出在哪

* edit: 我這邊的問題出在剛更新的文本不小心指令碼出錯，所以跟你遇到的問題無關。那我不確定你是遇到什麼問題了

**Comment by GpointChen** [comment]
我上傳了我的build：https://github.com/GpointChen/SaintCoinach/releases

你可以試試看。

**Comment by mokeling** [comment]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/56252068/0d894b26-13e8-4570-8fbd-ac130328e078)
很抱歉忘记回复了，依旧报错，甚至还多了一个（）  
有另外的大大和我说是因为没有更新表定义，[这是](https://github.com/nyaoouo/FFDraw/tree/master/fpt4/utils/sqpack/pack)她的写的解包程序但是我不会用

**Reply by GpointChen** [reply]
我在剛更新的release中附上了漢化後的六個檔案

如果你還沒解決可以用那個測試看看

**Reply by mokeling** [reply]
依旧报错，但是当我用繁体汉化时就解包正常
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/56252068/2c979e18-4e31-40d3-8015-bc4d6d43eb48)
所以现在的情况就是，解包中国服正常，解包繁体汉化正常，只有简体汉化会显示length错误  
不清楚是不是pc运行环境的问题，不过没关系了，我拿繁体简化一下就行了，谢谢大大了

**Reply by GpointChen** [reply]
了解，那可能是繁轉簡時有些詞彙字數會變動，導致指令碼出錯？雖然我也不知道為什麼我之前用簡體輸出時是正常的。

不過能用就好。
